components/pivotgrid/templates.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Templates
+page_title: PivotGrid Templates
+description: Learn about the templates of the Telerik UI for Blazor PivotGrid.
+slug: pivotgrid-templates
+tags: telerik,blazor,pivotgrid
+published: True
+position: 30
+---
+
+# PivotGrid Templates
+
+This article describes the PivotGrid templates. They allow you to customize the content and appearance of the PivotGrid row headers, column headers and data cells.
+
+* [`ColumnHeaderTemplate`](#column-header-template)
+* [`DataCellTemplate`](#data-cell-template)
+* [`RowHeaderTemplate`](#row-header-template)
+* [Example](#example)
+
+
+## Column Header Template
```

Comment:
I think it will be useful to mention that the templates are declared on component level and not per row/column instance.